public/app/plugins/datasource/prometheus/querybuilder/components/PromQueryBuilder.tsx

```diff
@@ -133,6 +133,13 @@ async function getMetrics(
     await datasource.languageProvider.loadMetricsMetadata();
   }
 
+  // Error handling for https://github.com/grafana/support-escalations/issues/2985
```

Comment:
Let's not put a links to our escalations into OSS code. Those links are not accessible for most people looking at it.

Reply:
Got it! Thank you, removed the link to the escalations.